app/src/main/java/net/squanchy/search/SearchActivity.kt

```diff
@@ -189,8 +189,8 @@ class SearchActivity : AppCompatActivity(), SearchRecyclerView.OnSearchResultCli
         }
     }
 
-    override fun onSaveInstanceState(outState: Bundle?) {
-        outState?.putString(QUERY_KEY, searchField.text?.toString())
+    override fun onSaveInstanceState(outState: Bundle) {
```

Comment:
❌ no, this **is** nullable. You need to _create_ it yourself if you get passed down null!

This will crash.

Reply:
uhm ok, should I also change 
https://github.com/squanchy-dev/squanchy-android/blob/develop/app/src/main/java/net/squanchy/home/HomeActivity.kt#L193
and
https://github.com/squanchy-dev/squanchy-android/blob/develop/app/src/main/java/net/squanchy/schedule/tracksfilter/ScheduleTracksFilter.kt#L178
?